### PR TITLE
Add UTU token metadata

### PR DESCRIPTION
**Pull request by databu**
# Add UTU token metadata

ERC-20: https://etherscan.io/address/0xa58a4f5c4Bb043d2CC1E170613B74e767c94189B
NEP-141: https://explorer.mainnet.near.org/accounts/a58a4f5c4bb043d2cc1e170613b74e767c94189b.factory.bridge.near
Aurora ERC-20 (BlockScout): https://explorer.mainnet.aurora.dev/address/0x7aa1a72f744cdcd4c89918aebfbe4f1d1d1156e6
Aurora ERC-20 (Aurorascan): https://aurorascan.dev/address/0x7aa1a72f744cdcd4c89918aebfbe4f1d1d1156e6

### CODE REVIEW

File: tokens/utu.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...IsMCwwLDAsODAuMzksNTZsLTEsOC41LDEuNzcuMiwxLTguNDloLjFBOC41Myw4LjUzLDAsMCwwLDg2LDQwbDEuODUtNS4zNGE2LjU3LDYuNTcsMCwwLDAsNS44Ny0uNDdsNC4zOSwzLjc3YTguNDUsOC40NSwwLDEsMCwxLjU5LTFMOTUuMSwzM2E2LjQ4LDYuNDgsMCwwLDAsMS43MS00Ljc2LDYuNTIsNi41MiwwLDAsMC0xMi41NS0yLjEzbC04LjgzLjE2YTEzLDEzLDAsMSwwLTE3LjQsMTVsLTMuMzEsOC44OWE4LjU0LDguNTQsMCwwLDAtOC4wOCw4LjUxLDcuOTQsNy45NCwwLDAsMCwwLC44NUwzOS42LDYxLjg3YTEzLDEzLDAsMCwwLTEwLTQuNjhoLS4yNkwyOC43Nyw1MmE2LjUxLDYuNTEsMCwwLDAsNC43Ni01LjQ1bDQuMTMtLjA5YTUuNTIsNS41MiwwLDEsMCw1LjgyLTYuMjZMNDMsMzUuNzlhNS41Myw1LjUzLDAsMSwwLTIsLjMzbC40Niw0LjI2YTUuNTMsNS41MywwLDAsMC0zLjc0LDRsLTQuMjcuMDlhNi41Miw2LjUyLDAsMSwwLTYuNzEsNy42N2wuNTYsNS4yMWExMywxMywwLDEsMCwxMy40OCw2LjJsNi4zLTIuMDhhOC41OCw4LjU4LDAsMCwwLDUuMzEsNS4yNmwtLjY2LDguNWMtMS43OC0uMTYtMy43NC4yOS02LjA4LDMuMzdhNSw1LDAsMCwwLTEsMi43MUE2LjQ5LDYuNDksMCwwLDAsNDcsODYuNzZsLTMuNzEsMy41YTksOSwwLDEsMCwxLjI3LDEuMjVabTM1LjI0LTU5LjhjMCwuMjIsMCwuNDUsMCwuNjlhNi41LDYuNSwwLDAsMCwyLjQ5LDUuMTFsLTIsNS42NmE4LjUsOC41LDAsMCwwLTcuNSwxLjczbC00LjcxLTNhMTMsMTMsMCwwLDAsMy42NS05YzAtLjM2LDAtLjcsMC0xLjA1WiIvPjxwYXRoIGNsYXNzPSJjbHMtMyIgZD0iTTc5Ljc2LDYwLjJhMTguMDUsMTguMDUsMCwxLDEtMTguMDYsMTgsMTguMDYsMTguMDYsMCwwLDEsMTguMDYtMThaIi8+PC9nPjwvZz48L3N2Zz4=",
```
Comment (karim-en) [inline tokens/utu.json:7]: There are missing fields eg  "near_address" & "icon"

**Review by karim-en — CHANGES_REQUESTED** [review]

File: tokens/utu.json (hunk 1 above)
Comment (Karkunow) [reply tokens/utu.json:7]: @databu will you update this one?

Comment (databu) [reply tokens/utu.json:7]: Hi, yes, sorry, it slipped my attention. I'm afk this week and next, will update after.

Comment (databu) [reply tokens/utu.json:7]: Hi, yes, sorry, it slipped my attention. I'm afk this week and next, will update after.

Comment (databu) [reply tokens/utu.json:7]: Hi, yes, sorry, it slipped my attention. I'm afk this week and next, will update after.

**Pull request closed without merging**